buildSrc/src/main/kotlin/toolkit-publishing-conventions.gradle.kts

```diff
@@ -54,7 +54,8 @@ configurations {
 
     // Make sure we exclude stuff we either A) ships with IDE, B) we don't use to cut down on size
     runtimeClasspath {
-        exclude(group = "org.slf4j")
+        exclude(group = "com.google.code.gson")
```

Comment:
This package is included only in certain environments right but we exclude it in all?

Reply:
yeah it does not hurt